src/components/link.js

```diff
@@ -171,6 +171,7 @@ module.exports.Component = registerComponent('link', {
     var cameraWorldPosition = new THREE.Vector3();
     var elWorldPosition = new THREE.Vector3();
     var quaternion = new THREE.Quaternion();
+    var quaternionClone = new THREE.Quaternion();
```

Comment:
This should be on the instance of the component, else all links will share this. It seems `this.previousQuaternion` is meant to be stateful.

Reply:
Yes, the variable is stateful.